.github/workflows/build.yml

```diff
@@ -69,39 +69,39 @@ jobs:
           fi
           google-chrome --version
 
-      # https://github.com/actions/cache/blob/main/examples.md#node---yarn
-      - name: Get Yarn cache directory
-        id: yarn-cache-dir-path
-        run: echo "dir=$(yarn cache dir)" >> $GITHUB_OUTPUT
-      - name: Cache Yarn dependencies
+      # https://github.com/actions/cache/blob/main/examples.md#node---npm
+      - name: Get NPM cache directory
+        id: npm-cache-dir
+        run: echo "dir=$(npm config get cache)" >> $GITHUB_OUTPUT
+      - name: Cache NPM dependencies
         uses: actions/cache@v4
         with:
-          # Cache entire Yarn cache directory (see previous step)
-          path: ${{ steps.yarn-cache-dir-path.outputs.dir }}
-          # Cache key is hash of yarn.lock. Therefore changes to yarn.lock will invalidate cache
-          key: ${{ runner.os }}-yarn-${{ hashFiles('**/yarn.lock') }}
-          restore-keys: ${{ runner.os }}-yarn-
+          # Cache entire NPM cache directory (see previous step)
+          path: ${{ steps.npm-cache-dir.outputs.dir }}
+          # Cache key is hash of package-lock.json. Therefore changes to package-lock.json will invalidate cache
+          key: ${{ runner.os }}-npm-${{ hashFiles('**/package-lock.json') }}
+          restore-keys: ${{ runner.os }}-npm-
 
-      - name: Install Yarn dependencies
-        run: yarn install --frozen-lockfile
+      - name: Install NPM dependencies
+        run: npm clean-install
 
       - name: Build lint plugins
-        run: yarn run build:lint
+        run: npm run build:lint
 
       - name: Run lint plugin tests
-        run: yarn run test:lint:nobuild
+        run: npm run test:lint:nobuild
 
       - name: Run lint
-        run: yarn run lint:nobuild --quiet
+        run: npm run lint:nobuild --quiet
```

Comment:
This is [not actually running in quiet mode](https://github.com/DSpace/dspace-angular/actions/runs/10718389527/job/29720201273?pr=3173#step:10:25). With npm, to pass arguments that start with a dash to the script, you need to [prefix them with `--`](https://stackoverflow.com/a/14404223) so this should be `npm run lint:nobuild -- --quiet`

Reply:
Good eye. I caught another similar occurrence of this in the Dockerfile.